site/content/recipes/meet_and_agree_on_objectives.md

```diff
@@ -0,0 +1,76 @@
+# Objective
+
+Set the code review up for success by making sure the reviewer and author are on the same page.
+
+# Steps
+
+-  If they have some, reviewer(s) state their expectation for the code review.
+-  If they have some, the code author state their expectation for the code review.
+-  Author and reviewers agree on the focus of the review
+
+# Description
+
+Code review can be a powerful tool to improve code quality&#x2026; provided
+that participants agree on what "improve" means. Should you focus on
+code readability? Or performance? For a code review to be productive,
+both the author and the reviewer(s) must agree on the direction to
+follow.
+
+During the startup meeting, all participants in the review state their
+expectations and agree on what the objective of the review should be.
+As the code author, you could plan to receive feedback on a particular
+design choice you made last week.  Or maybe you're planning to pick
+the reviewer's brain about optimising the performance of that sluggish
+subroutine. As a reviewer, you could be motivated by learning new
+features of a programming language. You could also want to put the
+focus on readability, because you think there is room for improvement
+in this direction. These expectations must be made clear during the
+startup meeting, before the review itself starts.
+
+As a rule of thumb, we recommend that the review focuses on one of the
+following themes, ordered by priority:
+
+-   Code readability
+-   Code maintainability
+-   Code extensability
+-   Performance
+
+Particularly, we do not recommend focusing on performance unless it is
+justified from a scientific perspective. Optimising for performance
+often makes code harder to read and maintain.
+
+Another special case is a "debugging session". The code author is looking
+for help to fix a specific problem, and code quality may not be the primary
+focus of the code review.
+
+# Examples
+
+-   R is familiar with the basics of C++, but never used it
+    frequently. They would like to learn what kind of features their C++
+    programmers colleagues use, and learn a few tricks along the way.  A
+    has been working on a C++ analysis pipeline on their own for the
+    past few weeks and is about to share it with their team. Before they
+    do so, they would like to check that the code is easy to read and
+    the structure easy to understand, even by colleagues with little
+    experience in C++. R and A decide
```

Comment:
I think you didn't finish this sentence.

Reply:
Ah yes I need to fix that, good catch

Reply:
Consider making these examples "vignettes" and having a visual offset for each so the reader can see that each is self-contained (and can be skipped if that's appropriate).